docs/algorithms.md

```diff
@@ -477,6 +477,16 @@ Terminal recording session
 
 For real-world examples to test with cocos, see our [AI repository](https://github.com/ultravioletrs/ai).
 
+## Running Python Algorithms with arguments
+
+To run a python algo that requires command line arguments, you can append the algo command on cli with the arguments needed as shown in the addition example:
+
+```bash
+./build/cocos-cli algo ./test/manual/algo/addition.py ./private.pem -a python --args="--a" --args="100" --args="--b" --args="20"
```

Comment:
include more detail on how to add arguments, order, how may you need to add, show how the added args will be passed by agent to the algo